src/components/search/SuggestedProductCard/suggested-product-card.scss

```diff
@@ -0,0 +1,50 @@
+.suggested-product-card {
+  padding: var(--space-1) var(--space-3);
+
+  [data-card-content] {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    align-items: center;
+  }
+
+  [data-card-image] {
+    margin-right: var(--space-3);
+  }
+
+  [data-gatsby-image-wrapper] {
+    flex-shrink: 0;
+    border-radius: var(--border-radius-default);
+  }
+
+  [data-suggested-product-card-summary] {
+    flex-direction: column;
+    grid-column: 2 / span 4;
+
+    [data-suggested-product-card-title] {
+      display: -webkit-box;
+      display: -moz-box;
+      display: box;
```

Comment:
I think the display box can generate some problems, we did a[ PR to fix a similar problem ](https://github.com/vtex-sites/base.store/pull/354/files)